src/core/main.cpp

```diff
@@ -528,6 +528,57 @@ int main( int argc, char * * argv )
 		}
 	}
 
+	// Test file argument before continuing
+	QFileInfo fileInfoLoad( fileToLoad );
+	QFileInfo fileInfoImport( fileToImport );
+	if( !fileToLoad.isEmpty() )
```

Comment:
Shouldn't this be the other way around?  Shouldn't we only instantiate `fileInfoLoad` if the QString isn't null or empty?  This would also remove the need for setting an empty QString above.